frame/dex-router/src/lib.rs

```diff
@@ -216,22 +355,43 @@ pub mod pallet {
 			Ok(())
 		}
 
-		fn get_route(asset_pair: CurrencyPair<T::AssetId>) -> Option<Vec<T::PoolId>> {
-			DexRoutes::<T>::get(asset_pair.base, asset_pair.quote)
-				.map(|DexRoute::Direct(route)| route.into_inner())
+		/// Returns pair of route and bool to indicate if route should be used in reverse direction
```

Comment:
Consider defining an enum for `Direction` and returning `Direction.Reverse` for reverse routes.

Reply:
not helpful, in current code I can check boolean in if directly which is much simpler compare to enum.